app/ui/legacy/src/main/res/values/strings.xml

```diff
@@ -1034,7 +1034,7 @@
 
     <string name="dialog_openkeychain_info_title">No OpenPGP app installed</string>
     <string name="dialog_openkeychain_info_install">Install</string>
-    <string name="dialog_openkeychain_info_text">K-9 Mail requires OpenKeychain for end-to-end encryption.</string>
+    <string name="dialog_openkeychain_info_text">The app requires OpenKeychain for end-to-end encryption.</string>
```

Comment:
If we're changing this anyway, we probably want to switch to passive voice and use "the app" to provide more context for people who don't know what "OpenKeychain" is.

```suggestion
    <string name="dialog_openkeychain_info_text">The app OpenKeychain is required to enable support for end-to-end encryption.</string>
```